lib/food_truck.rb

```diff
@@ -0,0 +1,27 @@
+class FoodTruck
+  attr_reader :name, :inventory
+
+  def initialize(name)
+    @name = name
+    @inventory = {}
+  end
+
+  def check_stock(item)
+    inventory[item] ||= 0
+  end
+
+  def stock(item, quantity)
+    inventory[item] ||= 0
+    inventory[item] += quantity
+  end
+
+  #all item prices times the quantity of items
+  # quantity * price = pr
+  # sum all pr together
+  def potential_revenue
+    @inventory.map do |invo|
+    invo.price * invo.quantity
+    end
```

Comment:
Close, but we'd need either a different enumerable or one more step to this enumerable. This method, as it's written right now, would return an array and inside the array would be elements that are the result of invo.price * invo.quantity. But then we need to sum all those together. So either need a `.sum` at the end of this method, or use the `.sum` enumerable instead of `.map`